weave

```diff
@@ -1972,20 +1981,10 @@ EOF
             shift
         done
         [ -n "$SUB_STATUS" ] || echo
-        if status=$(call_weave GET $STATUS_URL) ; then
-            case "$status" in
-                "")
-                    ;;
-                "404 page not found")
-                    echo "Invalid 'weave status' sub-command: $SUB_COMMAND" >&2
-                    usage
-                    ;;
-                *)
-                    echo "$status"
-                    ;;
-            esac
-        else
-            res=1
+        call_weave GET $STATUS_URL || res=$?
+        if [ $res = 404 ] ; then
```

Comment:
Just to underline the "undefined" nature, `weave status foo` gives usage as expected when run with Ubuntu's `/bin/sh` but not with Alpine's